pkg/crc/machine/bundle/metadata.go

```diff
@@ -355,13 +355,13 @@ func Download(preset crcPreset.Preset, bundleURI string) (string, error) {
 		if preset == crcPreset.OpenShift || preset == crcPreset.Microshift {
 			return DownloadDefault(preset)
 		}
-		return image.PullBundle(preset, "")
+		return image.PullBundle(constants.GetDefaultBundleImageRegistryURL(preset))
```

Comment:
I'd rewrite this as a `switch/case`, but I can add this to https://github.com/crc-org/crc/pull/3813